config/plaintext.yml

```diff
@@ -0,0 +1,38 @@
+# Text extraction helper programs.
```

Comment:
As the plaintext.yml was added to gitignore, it should probably not have been checked in.

Reply:
Good catch. Thanks!